packages/frontend/src/pages/IndicatorDetailPage.tsx

```
@@ -0,0 +1,87 @@
+        <IndicatorMetricsTable metrics={data.metrics} />
+        <IndicatorAnalyticsPanel metrics={{
+          sma50: data.analyticalMetrics?.latestSma50,
+          sma200: data.analyticalMetrics?.latestSma200,
+          yearlyHigh: data.analyticalMetrics?.yearlyHigh,
+          yearlyLow: data.analyticalMetrics?.yearlyLow,
+        }} />
+      </div>
+      {data.seriesInfo?.notes && (
+        <div className="p-4 bg-card text-card-foreground rounded-lg shadow mt-6">
+          <h3 className="text-lg font-semibold mb-2 text-primary">Notes</h3>
+          <div className="text-sm prose prose-sm max-w-none" dangerouslySetInnerHTML={{ __html: data.seriesInfo.notes }} />
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)